packages/react-ui-core/src/Menu/__tests__/Menu-test.js

```diff
@@ -35,6 +35,8 @@ const objectOptionsWithLabel = [
   },
   {
     label: 'Baz label',
+    value: 'Baz Value',
+    disabled: true,
```

Comment:
Do we need some tests for validating what happens when you disable something?

Reply:
Yes!

Reply:
There is a test for that already but the description might not be clear enough

Reply:
Fixed

Reply:
Okay, to be clear, he wrote tests in the PR from 2 days ago but had issues with that resolution and in this PR he is fixing stuff but the tests from before remain.